pkg/mount/scuzz/ata.go

```diff
@@ -148,8 +148,9 @@ func unpackIdentify(s statusBlock, d dataBlock, w wordBlock) *Info {
 	info.FirmwareRevision = ataString(d[46:54])
 	info.Model = ataString(d[54:94])
 
-	info.MasterPasswordRev = w[92]
-	info.SecurityStatus = w[128]
+	info.MasterPasswordRev = binary.LittleEndian.Uint16(d[184:186])
```

Comment:
there's some.... internal v1 use of this that I would like you to test before doing this as well

Reply:
ran all tests and confirmed that they pass!